docs/api/actions-and-connectors/create.asciidoc

```diff
@@ -28,6 +28,8 @@ Creates an action.
   (Required, object) The secrets configuration for the action. Secrets configuration properties vary
   depending on the action type. For information about the secrets configuration properties, refer to <<action-types,Action and connector types>>.
 
+WARNING: Remember these values. You must provide them each time you call the <<actions-and-connectors-api-update, update>> API.
```

Comment:
That's correct because it is specific to `secrets. Adding `+` to the line above worked, thanks! Fixed in https://github.com/elastic/kibana/pull/92411/commits/b24a4b1411ff02ba3d288bf551a4841c12b8e63c ready for re-review.